lavalab-gen.py

```diff
@@ -74,18 +74,20 @@
 """)
 
 def main():
+    need_zmq_auth_gen = False
     fp = open(boards_yaml, "r")
     workers = yaml.load(fp)
     fp.close()
 
     os.mkdir("output")
+    zmq_auth_genlist = open("zmqauth/zmq_auth_gen/zmq_genlist", 'w')
```

Comment:
Why is this here?  shouldn't it be later and conditional based on ZMQ usage?

Reply:
It's a list of "for which host we need to generate a key" and so it must be reseted early.
I could put it later, but I would need to detect "does it was already reseted ?" and so make it more complex.

Reply:
ok, I see.